env/default/default.properties

```diff
@@ -49,7 +49,7 @@ OPERATOR_VERSION = devel
 OSP_VERSION = 5.0.5
 OSP_TUTORIAL_BRANCH = master
 PAC_VERSION = 0.30
```

Comment:
Change also this to 0.32, see https://github.com/tektoncd/operator/blob/main/components.yaml#L16-L18

Reply:
Done